cardano_node_tests/tests/test_plutus_delegation.py

```diff
@@ -363,6 +364,9 @@ def test_delegate_deregister(
             pool_id=pool_id,
         )
 
+        # check tx_view
+        tx_view.check_tx_view(cluster_obj=cluster, tx_raw_output=tx_raw_delegation_out)
+
```

Comment:
Please move this non-essential check to the end of the test function (next to the other `check_tx_view`)